Sources/DepoCore/Shell/SwiftPackageShellCommand.swift

```diff
@@ -163,9 +178,9 @@ public final class SwiftPackageShellCommand: ShellCommand {
         }
     }
 
-    private func upToVersionConstraint(from requirement: JsonerOutputWrapper.Dependency.Requirement) -> VersionConstraint<SwiftPackage.Operator>? {
-        guard let lowerBound = requirement.lowerBound,
-              let upperBound = requirement.upperBound,
+    private func upToVersionConstraint(from req: JsonerOutputWrapper.Dependency.Requirement) -> VersionConstraint<SwiftPackage.Operator>? {
+        guard let lowerBound = req.lowerBound,
```

Comment:
why do you rename it to `req` instead of `requirement`? according [Code Style](https://github.com/rosberry/swift-style-guide) we should avoid abbreviations

Reply:
It's req here to complete the code-style line-length requirement

Reply:
you can use typealiases for long types